pkg/cloud/aws/kinesis/kinsumer.go

```diff
@@ -135,6 +138,7 @@ func NewKinsumer(ctx context.Context, config cfg.Config, logger log.Logger, sett
 	}
 
 	shardReaderFactory := func(logger log.Logger, shardId ShardId) ShardReader {
+		logger = logger.WithChannel("kinsumer-read")
```

Comment:
I would argue this call belongs to the `NewShardReaderWithInterfaces` call and not here